ckanext/canada/tables/experiment.yaml

```diff
@@ -0,0 +1,396 @@
+          fr: "Question de recherche (en anglais)"
+        obligation: Mandatory
+        occurrence: Single
+        validation: "This field must not be empty"
+        visible_to_public: true
+
+      - datastore_id: question_de_recherche_fr
+        datastore_type: text
+        description:
+          en: "This field describes the research question informing the project, in French"
+          fr: "Ce champ décrira la question de recherche guidant le projet, en français"
+        excel_column_width: 28
+        excel_required: true
+        format_type: "Free text"
+        label:
+          en: "Research Question (French)"
+          fr: "Question de recherche (en français)"
+        obligation: Mandatory
+        occurrence: Single
+        validation: "This field must not be empty"
+        visible_to_public: true
+
+      - datastore_id: project_summary_en
+        datastore_type: text
+        description:
+          en: "This field summarizes the key information about the project, including any findings once the initiatve is completed, in English"
+          fr: "Ce champ résumera les information-clés à propos du projet, y compris les résultats lorque le projet est complété, en anglais"
+        excel_column_width: 35
+        excel_required: true
+        format_type: "Free text"
+        label:
+          en: "Project Summary (English)"
+          fr: "Résumé du projet (en anglais)"
+        obligation: Mandatory
+        occurrence: Single
+        validation: "This field must not be empty"
+        visible_to_public: true
+
+      - datastore_id: project_summary_fr
+        datastore_type: text
+        description:
+          en: "This field summarizes the key information about the project, including any findings once the initiatve is completed, in French"
+          fr: "Ce champ résumera les information-clés à propos du projet, y compris les résultats lorque le projet est complété, en français"
+        excel_column_width: 35
+        excel_required: true
+        format_type: "Free text"
+        label:
+          en: "Project Summary (French)"
+          fr: "Résumé du projet (en français)"
+        obligation: Mandatory
+        occurrence: Single
+        validation: "This field must not be empty"
+        visible_to_public: true
+
+      - datastore_id: last_updated
+        datastore_type: date
+        description:
+          en: "This field will display the latest update provided for the project."
+          fr: "Ce champ décrira à quand remonte la dernière mise à jour."
+        excel_column_width: 28
+        excel_required: true
+        format_type: "Date (Please format the data as YYYY-MM-DD)"
+        label:
+          en: "Last Updated"
+          fr: "Dernière mise à jour"
+        obligation: Mandatory
+        occurrence: Single
+        validation: "This field must not be empty"
+        visible_to_public: true
+
+      - datastore_id: experimental_area
+        choices:
+          001:
```

Comment:
if these have to be numbers put them in quotes so that they're treated as strings instead of integer keys (allowed in python yaml but not json)

Ideally we should be using letter codes for these instead, 0-prefixed numeric codes are difficult to enter in excel

Reply:
same comment about the other choice lists. Otherwise this looks good to me!

Reply:
I agree - these are just the values that were in the DEP. I spoke to AH and got the Okay to change these to alphabetic code values.

Reply:
Actually, I need your help with setting the Excel colors. I thought I was setting it to orange but the template came out blue.

Reply:
should be AARRGGBB format

Reply:
Thanks, it worked. I have made the code value changes.